packages/suite/src/actions/firmware/firmwareActions.ts

```diff
@@ -89,7 +89,7 @@ export const firmwareUpdate = () => async (dispatch: Dispatch, getState: GetStat
 
     const updateResponse = await TrezorConnect.firmwareUpdate(payload);
 
-    dispatch(
+    await dispatch(
```

Comment:
here it does not make sense to wait for http request completion, this would degrade user experience.